### ISSUE TITLE
run statistics-volume-correction raises "AssertionError: pattern argument cannot start with char: /"

### ISSUE THREAD

**Issue by AudreyDuran** [aramis-lab/clinica#827]
**Describe the bug**
Running `statistics-volume-correction` raises an `AssertionError`.

**To Reproduce**
1. run `statistics-volume` pipeline ; 
2. run the volume-correction pipeline: 
`clinica run statistics-volume-correction clinica_CAPS clinica_CAPS/groups/group-treatedvsplacebo/statistics_volume/group_comparison_measure-graymatter/group-treatedvsplacebo_treated-lt-placebo_measure-graymatter_fwhm-8_TStatistics.nii 3.1 5.273 6.059 25 16`

**Expected behavior**
The pipeline should run without errors

**Screenshots**

Raises the following error:
```bash
Traceback (most recent call last):
  File "/workspace/clinicaEnv/bin/clinica", line 8, in <module>
    sys.exit(main())
  File "/workspace/clinicaEnv/lib/python3.8/site-packages/clinica/cmdline.py", line 89, in main
    cli()
  File "/workspace/clinicaEnv/lib/python3.8/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/workspace/clinicaEnv/lib/python3.8/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/workspace/clinicaEnv/lib/python3.8/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/workspace/clinicaEnv/lib/python3.8/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/workspace/clinicaEnv/lib/python3.8/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/workspace/clinicaEnv/lib/python3.8/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/workspace/clinicaEnv/lib/python3.8/site-packages/clinica/pipelines/statistics_volume_correction/statistics_volume_correction_cli.py", line 83, in cli
    else pipeline.run()
  File "/workspace/clinicaEnv/lib/python3.8/site-packages/clinica/pipelines/engine.py", line 261, in run
    self.build()
  File "/workspace/clinicaEnv/lib/python3.8/site-packages/clinica/pipelines/engine.py", line 25, in func_wrapper
    res = func(self, *args, **kwargs)
  File "/workspace/clinicaEnv/lib/python3.8/site-packages/clinica/pipelines/engine.py", line 231, in build
    self.build_input_node()
  File "/workspace/clinicaEnv/lib/python3.8/site-packages/clinica/pipelines/statistics_volume_correction/statistics_volume_correction_pipeline.py", line 37, in build_input_node
    t_map = clinica_group_reader(
  File "/workspace/clinicaEnv/lib/python3.8/site-packages/clinica/utils/inputs.py", line 478, in clinica_group_reader
    assert pattern[0] != "/", (
AssertionError: pattern argument cannot start with char: / (does not work in os.path.join function). If you want to indicate the exact name of the file, use the format directory_name/filename.extension or filename.extension in the pattern argument.
```

The "pattern" is in this case the `T_MAP` ([source code](https://github.com/aramis-lab/clinica/blob/444e4a4e711d6a373fae07412fa378da1e356572/clinica/pipelines/statistics_volume_correction/statistics_volume_correction_pipeline.py#L40)), so here `clinica_CAPS/groups/group-treatedvsplacebo/statistics_volume/group_comparison_measure-graymatter/group-treatedvsplacebo_treated-lt-placebo_measure-graymatter_fwhm-8_TStatistics.nii`.

If I remove the path to the file to have a filename.extension format (as stated in the error message), I get the following error:
`Error: Invalid value for 'T_MAP': Path 'group-treatedvsplacebo_treated-lt-placebo_measure-graymatter_fwhm-8_TStatistics.nii' does not exist.`

I think that a step extracting the pattern from the path to the file is missing (or my usage is not correct, in that case sorry for that!).

**Desktop (please complete the following information):**
 - OS: linux
 - Browser: chrome
 - Clinica version 0.7.2

**Comment by ghisvail** [comment]
> Clinica version 0.7.2

Could you upgrade to version `0.7.3` and re-run the pipeline. This section of the code has been updated since, so the stack trace does not make much sense on current code.

**Comment by AudreyDuran** [comment]
Done, I still have the same error

```bash
(clinicaEnv) user@:/workspace# pip list | grep clinica
clinica                  0.7.3
(clinicaEnv) user@:/workspace# clinica run statistics-volume-correction clinica_CAPS/ clinica_CAPS/groups/group-treatedvsplacebo/statistics_volume/group_comparison_measure-graymatter/group-treatedvsplacebo_treated-lt-placebo_measure-graymatter_fwhm-8_TStatistics.nii 3.1 5.273 6.059 25 16
221206-08:09:12,105 nipype.utils WARNING:
         A newer version (1.8.4) of nipy/nipype is available. You are using 1.8.2

Traceback (most recent call last):
  File "/workspace/clinicaEnv/bin/clinica", line 8, in <module>
    sys.exit(main())
  File "/workspace/clinicaEnv/lib/python3.8/site-packages/clinica/cmdline.py", line 92, in main
    cli()
  File "/workspace/clinicaEnv/lib/python3.8/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/workspace/clinicaEnv/lib/python3.8/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/workspace/clinicaEnv/lib/python3.8/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/workspace/clinicaEnv/lib/python3.8/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/workspace/clinicaEnv/lib/python3.8/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/workspace/clinicaEnv/lib/python3.8/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/workspace/clinicaEnv/lib/python3.8/site-packages/clinica/pipelines/statistics_volume_correction/statistics_volume_correction_cli.py", line 84, in cli
    else pipeline.run()
  File "/workspace/clinicaEnv/lib/python3.8/site-packages/clinica/pipelines/engine.py", line 311, in run
    self.build()
  File "/workspace/clinicaEnv/lib/python3.8/site-packages/clinica/pipelines/engine.py", line 26, in func_wrapper
    res = func(self, *args, **kwargs)
  File "/workspace/clinicaEnv/lib/python3.8/site-packages/clinica/pipelines/engine.py", line 281, in build
    self.build_input_node()
  File "/workspace/clinicaEnv/lib/python3.8/site-packages/clinica/pipelines/statistics_volume_correction/statistics_volume_correction_pipeline.py", line 37, in build_input_node
    t_map = clinica_group_reader(
  File "/workspace/clinicaEnv/lib/python3.8/site-packages/clinica/utils/inputs.py", line 663, in clinica_group_reader
    _check_information(information)
  File "/workspace/clinicaEnv/lib/python3.8/site-packages/clinica/utils/inputs.py", line 311, in _check_information
    raise ValueError(
ValueError: pattern argument cannot start with char: / (does not work in os.path.join function). If you want to indicate the exact name of the file, use the format directory_name/filename.extension or filename.extension in the pattern argument.
```

**Comment by AudreyDuran** [comment]
@ghisvail any updates on this issue ?

**Comment by NicolasGensollen** [comment]
Closing as resolved by #858
